nodes/PayloadValidator.js

```diff
@@ -41,40 +42,76 @@ module.exports = class PayloadValidator {
     }
   }
 
+  /**
+   * Gets the value at the given location in the given object
+   *
+   * @param {*} object
+   * @param {*} location
+   * @returns
+   */
   getValue(object, location) {
-    return location.split('.').reduce((p, c) => (p && p[c]) || null, object);
+    return _.get(object, location);
+  }
+
+  /**
+   * Sets the value at the given location in the given object to the given value
+   *
+   * @param {*} object
+   * @param {*} location
+   * @param {*} value
+   * @returns
+   */
+  setValue(object, location, value) {
+    return _.set(object, location, value);
+  }
+
+  check(_after) {
```

Comment:
this method needs a name more suited to what it is doing and i think the method itself needs to be split up